### ISSUE TITLE
quesions about dataset

### ISSUE THREAD

**Issue by jiangduwang** [megvii-research/Iter-E2EDET#23]
hi,author!
The data set link you provided includes three training sets, one test set and one verification set. How do you put these files under the CrowdHuman folder？
Thanks!

**Comment by yexiguafuqihao** [comment]
Please check [this](https://github.com/megvii-research/Iter-E2EDET/blob/main/detectron2/data/datasets/builtin.py) for the detail.

**Comment by peterhuang621** [comment]
Hi, 
I have tried all train sets provided in CrowdHuman website but it keeps throwing errors for missing pics. And I also find very hard to identify the instruction in the file you mentioned. From my aspect, all train sets would be fine for just training step.
Could you please give us a clear clue to going on the experiment?
Thanks so much.

**Comment by yexiguafuqihao** [comment]
The problem may be derived from the unfamiliarity with the detectron2. Here is the procedure to curate the CrowdHuman dataset for both training and testing.

1. please download both training and validation datasets, and mix the separated training or validation dataset. Such as merge CrowdHuman_train01, CrowdHuman_train02 and CrowdHuman_train03 into one CrowdHuman_train dataset, so does the validation set.
2. Change the directory file at [this](https://github.com/megvii-research/Iter-E2EDET/blob/c649a0b29be48e7253048a2c8efd6b28a2e3197b/projects/crowd-e2e-sparse-rcnn/config.py#L79) to ensure the project can find image and json files properly in your own system. Besides, [this](https://github.com/megvii-research/Iter-E2EDET/blob/c649a0b29be48e7253048a2c8efd6b28a2e3197b/detectron2/data/datasets/builtin.py#L53)  code describes how to register the CrowdHuman dataset in detectron2.
3. If a loading image error still occurs after the above procedure is completed, [this](https://github.com/megvii-research/Iter-E2EDET/blob/c649a0b29be48e7253048a2c8efd6b28a2e3197b/detectron2/data/dataset_mapper.py#L125) line decribe the way the project load an image. Your can come up with an approach to load image properly. Such as add another key/value in the train.json/ val.json to redirect the proper image file location in your disk.

**Comment by peterhuang621** [comment]
Really appreciate your very kind and patient answer, and it's quite clear now!
Thanks again for your help.
